### PR TITLE
Updated locale_de.properties language file (german)

### PR DESCRIPTION
**Pull request by w1tcherrr**
Fixed various grammatical and spelling mistakes. Made some adjustments to the colour-scheme (whilst keeping the colours mcMMO uses as it's standard colours). Changed some phrases for consistency. Added dots at the end of every sentence.

### CODE REVIEW

File: src/main/resources/locale/locale_de.properties [hunk 1]
```diff
@@ -106,76 +106,76 @@ Combat.BeastLoreHorseJumpStrength = &3Pferde-Sprungst\u00E4rke (&aMaximal {0} Bl
...
-Commands.Description.vampirism = Schalte Vampirismus an oder aus bzw. \u00E4ndere den Vampirismus Prozentsatz
-Commands.Description.xplock = Setze deine Erfahrungs-Leiste auf einen bestimmten Skill fest
-Commands.Description.xprate = \u00C4ndere die Erfahrungs Rate oder starte ein Erfahrungs Event
+Commands.Description.mcrank = Zeige das Skill-Ranking f\u00FCr einen Spieler.
+Commands.Description.mcrefresh = Aktualisiere alle Cooldowns.
+Commands.Description.mcremove = Entferne einen Benutzer aus der Datenbank.
+Commands.Description.mcscoreboard = Verwalte deine Skill-\u00DCbersicht.
+Commands.Description.mcstats = Zeige deine Skill-Level und Erfahrung.
+Commands.Description.mctop = Zeige die Skill-Bestenlisten.
+Commands.Description.mmocompat = Informationen darüber, ob mcMMO im Kompatibilitätsmodus oder voll funktionsfähig ist.
```
Comment (TheBusyBiscuit) [inline src/main/resources/locale/locale_de.properties:151]: ä is not properly encoded in this string.

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]
The translations are all very good! A big impeovement, just a slight encoding mistake of an umlaut in one line.

File: src/main/resources/locale/locale_de.properties (hunk 1 above)
Comment (w1tcherrr) [reply src/main/resources/locale/locale_de.properties:151]: Thank you, I actually missed some of the UTF-8-Encoding. I will fix that today!

Comment (w1tcherrr) [reply src/main/resources/locale/locale_de.properties]: I replaced all the missing Umlaute with their UTF-8 encoding (forgot about that, sorry). I never worked with Github before to be honest, but I think you should be able to see the new commit. :)

**Review by TheBusyBiscuit — APPROVED** [review]
A good improvement!

File: src/main/resources/locale/locale_de.properties (hunk 1 above)
Comment (TheBusyBiscuit) [reply src/main/resources/locale/locale_de.properties]: Yep, whenever you commit to the branch you opened the pull request with, your changes/commits will automatically update in the pull request too!
It's looking very good to me now - as a fellow native speaker.

**Comment by nossr50** [comment]
Thank you

**Pull request merged**